CI_YAML.md

```diff
@@ -54,6 +54,19 @@ targets:
        ["framework", "hostonly"]
 ```
 
+## Adding new targets
+
+All new targets should be added as `bringup: true` to ensure they do not block the tree.
```

Comment:
We want to enforce adding the target in staging pool first before moving it to the prod, even with `bringup: true`. Could you add a comment here? 
BTW: do we have plans to support staging targets in the .ci.yaml?

Reply:
My understanding of staging is that we want it for new testbeds, not necessarily new tests. Is there a doc with more details?

I can add support for `.staging_ci.yaml` with all the plumbing to generate these staging builders, but this doc is intended for developers.

Reply:
It will be for any new test, not only a new testbed.
Thanks. I will /cc you when that section is ready.

Reply:
Would the plan be to deprecate the `bringup` field then? For example, what about re-sharding tests. Are those new tests?

Reply:
No. The bringup will still be helpful for existing tests that are flaky. Only those new tests or old ones that have not been running for a while need to go through the staging pool.
Since the shard tests have been running, the re-sharding probably not need to follow.